webpack/base.js

```diff
@@ -8,6 +8,14 @@ let htmlWebpackPlugin = require('html-webpack-plugin'),
 manifest.description = process.env.npm_package_description;
 manifest.version = process.env.npm_package_version;
 
+if (process.env?.TARGET_PLATFORM === 'firefox') {
+    manifest.browser_specific_settings = {
+        gecko: {
+            id: "sendtokodi@firsttris.github.io"
+        }
+    };
+}
+
```

Comment:
See https://github.com/firsttris/chrome.sendtokodi/pull/3#discussion_r788364886
I'm open to other approaches, this is just the first thing that came to mind.